Zend/zend_attributes.h

```diff
@@ -116,6 +116,19 @@ static zend_always_inline zend_attribute *zend_add_class_constant_attribute(zend
 	return zend_add_attribute(&c->attributes, name, argc, flags, 0, 0);
 }
 
+static zend_always_inline zend_attribute *zend_mark_function_parameter_as_sensitive(const HashTable *table, char *func_name, uint32_t parameter)
+{
+	zend_function *func = zend_hash_str_find_ptr(table, func_name, strlen(func_name));
+	ZEND_ASSERT(func != NULL);
```

Comment:
Does this work for `disable_functions`, because such functions are [removed from the `function_table`](https://github.com/php/php-src/blob/master/Zend/zend_API.c#L3234-L3237)?

Reply:
Yes, it does. I'm setting the attribute in MINIT and I assume that `disable_functions` does its thing after MINIT finishes.

In any case I've added a test for that: https://github.com/php/php-src/pull/8352/commits/82c5570fac94c4530461c326049798b845e2803a

Reply:
Ah, right! Having a test for this is still a good idea. Thanks!